plantsaver/start.py

```diff
@@ -20,7 +21,7 @@
     print("Status code:"+plantsaver.status)
 
     # Check if water level is too dry and that the pump wasn't on for the past 15 minutes
-    if plantsaver.status_code == 1 and pump_count >= plantsaver.pump_delay * 6:
+    if plantsaver.status_code == 1 and pump_count >= 6:
```

Comment:
Is this intentional? Looks like removing the delay would set it to pumping every minute if the tick is 10 seconds, but even so would be better to keep the delay variable and tune that downwards rather than hard coding?

Reply:
Oohhh... I read that all wrong haha. I thought that I could force it to pump every 6 counts until it hit some kind of target moisture reading.

I think my approach is wrong. I'll take another look.